test/gov/service_configstate_test.sh

```diff
@@ -129,22 +137,20 @@ if [ $? -ne 0 ]; then
 	echo -e "${PREFIX} error suspending e2edev@somecomp.com/netspeed: $out"
     exit 2
 fi
+
 echo -e "${PREFIX} suspending the e2edev@somecomp.com/location service..."
-out=$(hzn service configstate suspend e2edev@somecomp.com https://bluehorizon.network/services/location -f)
-if [ $? -ne 0 ]; then
-	echo -e "${PREFIX} error suspending e2edev@somecomp.com/location: $out"
-    exit 2
-fi
+read -d '' sconfig <<EOF
+{
+   "url":         "https://bluehorizon.network/services/location",
+   "org":          "e2edev@somecomp.com",
+   "configState": "suspended"
+}
+EOF
+out=$(echo "$sconfig" | curl -sLX POST $CERT_VAR -u $USERDEV_ADMIN_AUTH -H "Content-Type: application/json" -H "Accept: application/json" --data @- ${EXCH_URL}/orgs/userdev/nodes/an12345/services_configstate | jq -r '.')
+echo -e "${PREFIX} $out"
```

Comment:
You need to check the http status code from the curl so you can fail the test if it's bad.

Reply:
done